src/screens/About/About.tsx

```diff
@@ -334,27 +337,44 @@ const About = ({ children }: { children: React.ReactNode }) => {
             — but this is the basic modeling approach we use.
           </Typography>
 
+          <Typography variant="h6" component="h6">
+            What does the dotted line on the “infection growth” graph mean?
+          </Typography>
+
+          <Typography variant="body1" component="p">
+            Because of how we weight our data (Gaussian smoothing), and because
+            of potential reporting delays and errors in the incoming case data,
+            we need 7 preceding days of data before we can calculate a final R
+            <sub>t</sub> value. Therefore, we notate preliminary R<sub>t</sub>{' '}
```

Comment:
Flag (feel free to ignore), I don't think we use R<sub>t</sub> anywhere else. We use R-effective above and in our blog post: https://blog.covidactnow.org/modeling-metrics-critical-to-reopen-safely/